compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1678,3 +1678,9 @@ error.invalid.readonly.field.type=\
 error.continue.not.allowed=continue not allowed here
 
 error.break.not.allowed=break not allowed here
+
+error.invalid.iterable.type=\
+  invalid iterable type ''{1}'': an iterable object must be a subtype of ''{0}''
```

Comment:
Do we have tests with these error messages?

Reply:
Added one 4d3bafa3ac1fffc61878c0261883119cbe5ad9bc